templates/components/products/carousel.html

```diff
@@ -6,8 +6,8 @@
         "slidesToShow": 2,
         "slidesToScroll": 1,
         "slide": ".js-product-slide",
-        "prevArrow": ".js-product-prev-arrow",
-        "nextArrow": ".js-product-next-arrow",
+        "prevArrow": ".js-product-prev-arrow{{#if list '===' "Customers Also Viewed"}}-viewedProducts{{/if}}",
+        "nextArrow": ".js-product-next-arrow{{#if list '===' "Customers Also Viewed"}}-viewedProducts{{/if}}",
```

Comment:
I believe we shouldn't use hardcode like this in reusable component. Could we find any better solution?

Reply:
we have this text as a param in template
<img width="977" alt="tab_paremeter_for_carousel" src="https://user-images.githubusercontent.com/66325265/92602948-6ffcd900-f2b7-11ea-801e-24f67e32cac2.png">
any ideas how to save it to variable?

Reply:
Could we use more specific selector for arrows with carousel specifying?

Reply:
Can you please add more details, can not understand the context. We have two instances of templates/component/product/carousel.html, and we need somehow to specify different arrow classes for them

Reply:
We could use something like additional field 'customArrowClass' or data-attribute but not create dependency with title value @yurytut1993

Reply:
I added new param customArrowClassAddition to update carousel arrow class and to make in unique

Reply:
From my view, better way to implement it - to add additional class, not addition. In case of additional css/js logic we need to rewrite current implementation

Reply:
I added separate param for arrow class, when this param is not setted, product carousel use its base class for arrows